api/db/migrations/20200623155650_add_updatedByUser_to_memberships.js

```diff
@@ -0,0 +1,14 @@
+const TABLE_NAME = 'memberships';
+const COLUMN_NAME = 'updatedByUserId';
```

Comment:
Je crois qu'on avait convenu que le nom de la colonne serait simplement `updatedByUser`

Reply:
Yes c'était ma proposition en plus mais au final je trouvais ca plus clair :)

Reply:
Mais du coup vous créez une colonne qui contient un user id mais qui est de type `string` ? 🤔

Reply:
Changed to table.integer(COLUMN_NAME).unsigned();

Reply:
Pourquoi ne pas indiquer que c'est une clé étrangère sur users.id ?

Reply:
👍 @chrc c’était ma question d’après 😉

Reply:
J'ai rajouté la foreign key 
`    table.integer(COLUMN_NAME).unsigned().references('id').inTable('users ');
`

Reply:
`table.integer(COLUMN_NAME).references('users.id');`

Reply:
Allez je prend ta syntaxe pour avoir de la consistence.